### DISCUSSION TITLE
RFP & CB together

### DISCUSSION THREAD

**Discussion by EoinDub** [kkapsner/CanvasBlocker#729] [General]
I know this has been asked before but I can't really see a consensus. Is it worth using resist finger printing and CB together? 

I had RFP turned off but noticed in the CB fingerprint tests, the offscreen test and offscreen worker tests don't change hash on reload of page, but the others do. Turning on RFP ensures it does. Would it be worth having both for edge cases like this? Am I just missing a setting in CB?

Wondering the pros and cons for using both, or is it overkill?

**Comment by kkapsner** [comment]
Yes - it is worth using both. RFP has use cases and CB also has. The workers are one example case for RFP.

But it has trade offs. RFP can be annoying and render web paged unusable if not disabled for them. CB then can jump in and still give protection.
